client/src/components/sellerName.jsx

```diff
@@ -3,19 +3,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-const SellerNameP = styled.p`
-  font-family: "Graphik Webfont",-apple-system,BlinkMacSystemFont,"Roboto","Droid Sans","Segoe UI","Helvetica",Arial,sans-serif;
-  font-weight: 300;
+const SellerNameA = styled.a`
+  color: black;
   font-size: 16px;
-  line-height: 28px;
+  text-decoration: none;
+  &: hover {
+    text-decoration: underline;
+  }
 `;
 
 function SellerName(props) {
   const { data } = props;
   return (
-    <SellerNameP>
+    <SellerNameA href="TODO">
```

Comment:
If you need to complete the href, try using "#TODO" instead

https://timwise.co.uk/2013/05/25/unfinished-hyperlinks-add-todo/#